shared/messaging/proto/events.proto

```diff
@@ -0,0 +1,7 @@
+syntax = "proto3";
+
+package pocket;
+
+option go_package = "github.com/pokt-network/pocket/shared/messaging";
+
+message NodeStartedEvent {}
```

Comment:
Thoughts on something like this?

```proto
enum NodeEvent {
	NODE_EVENT_UNDEFINED = 0;
	NODE_EVENT_STARTED = 1;
}

message NodeEvent {
     NodeEvent event = 1;
}
```

Reply:
I'd prefer individual events that have their own properties and/or enums if needed. 

This is just a "flagging event" with no properties so I get your point, however let's think about more complex events.

Lemme try to explain what I mean:

If we have a "catch-all" NodeEvent, we'd have to inspect the content on the handler side everytime, also you would end up having properties in there that make sense for an event but not for another. 
This increases the risk of breaking changes down the line and also complexity IMHO.

With individual types, instead, you just look at the envelope if that makes sense and you treat apples like apples and oranges like oranges.